corehq/apps/reports/tests/test_esaccessors.py

```diff
@@ -92,6 +94,8 @@ def _send_form_to_es(
             setattr(form_pair.wrapped_form, 'external_blobs', attachment_dict)
             form_pair.json_form['external_blobs'] = attachment_dict
 
+        if settings.TESTS_SHOULD_USE_SQL_BACKEND:
```

Comment:
👍along with transform_xform_for_elasticsearch.
This was part of a bigger PR where I moved add_couch_properties_to_sql_form_json inside transform_xform_for_elasticsearch which is probably why I didn't see the failure locally.